examples/recording_service/pluggable_storage/cpp/FileStorageReader.cxx

```diff
@@ -15,6 +15,10 @@
 #include "dds/dds.hpp"
 #include <limits>
 
+#ifdef RTI_WIN32
+    #undef max
```

Comment:
Wasn't there a `NOMINMAX` define that we could use or something similar?

Reply:
I think so, we can use either max and min or NOMINMAX. As I think that we will never use min on this scenario the  under max solution should be enough